samples/helm-deploy/values.yaml

```diff
@@ -0,0 +1,27 @@
+# Default values for carbon-aware-sdk.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+namespace: default
+
+image:
+  name: carbon-aware-sdk
+  repository: ghcr.io/green-software-foundation/carbon-aware-sdk
+  pullPolicy: IfNotPresent
+  tag: "latest"
+  auth:
+    username: username
+    password: password
```

Comment:
We should consider other parameters (e.g. data source) if we can set them via environment variables and/or ConfigMap.